src/main/java/com/company/GetTimeline.java

```diff
@@ -15,13 +17,9 @@ public static void main(String[] args) {
         TwitterFactory tf = new TwitterFactory(cb.build());
         Twitter twitter = tf.getInstance();
         try{
-            ResponseList<Status> homeTimeLine = twitter.getHomeTimeline();
-            for(Status s : homeTimeLine){
-                System.out.println("@"+s.getUser().getScreenName());
-                System.out.println(s.getText());
-            }
+            return twitter.getHomeTimeline();
         } catch(TwitterException e){
-            System.out.println("error getting twitter timeline");
+            return null;
```

Comment:
why are you returning null and eating the exception? what are you returning if the call fails?